### PR TITLE
Fix push command validation

### LINKED ISSUE #34851: Misleading error message for cli command
**Description:**
image(1)

As depicted above, the error message suggests that the `bala` file is erroneous instead of suggesting that the `command` that I have used to push the package is incomplete. (missing `--` before the argument) 

**Steps to reproduce:**
Mentioned above

**Affected Versions:**
sl-GA-rc2

### PR DESCRIPTION
**Pull request by Ibaqu**
## Purpose
Adds an extra validation for empty or invalid files provided to push command

Fixes #34851 

## Check List 
- [ ] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/PushCommand.java [hunk 1]
```diff
@@ -161,6 +162,10 @@ public void execute() {
                 if (balaPath == null) {
                     pushPackage(project);
                 } else {
+                    // If the file doesnt exist, and if the file is not a bala file
+                    if (!balaPath.toFile().exists() && !FileUtils.getExtension(balaPath).equals("bala")) {
```
Comment (azinneera) [inline cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/PushCommand.java:166]: Shall we check the missing file and invalid extension cases separately and throw appropriate error messages?

Comment (Ibaqu) [reply cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/PushCommand.java]: Checked the cases separately with the appropriate error messages

File: cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/PushCommand.java [hunk 2]
```diff
@@ -161,6 +162,12 @@ public void execute() {
                 if (balaPath == null) {
                     pushPackage(project);
                 } else {
+                    if (!balaPath.toFile().exists()) {
+                        throw new ProjectException("provided bala file does not exist.");
```
Comment (azinneera) [inline cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/PushCommand.java:166]:
```suggestion
                        throw new ProjectException("provided path does not exist.");
```

Comment (Ibaqu) [reply cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/PushCommand.java]: Changed to `path`

**Review by azinneera — APPROVED** [review]

**Pull request merged**